BubblesKotlin/app/src/main/java/com/example/android/bubbles/data/NotificationHelper.kt

```diff
@@ -154,6 +166,7 @@ class NotificationHelper(private val context: Context) {
             .setSmallIcon(R.drawable.ic_message)
             .setCategory(Notification.CATEGORY_MESSAGE)
             .setShortcutId(chat.contact.shortcutId)
+            .setLocusId(LocusId(chat.contact.shortcutId))
```

Comment:
Maybe add some comments for each line to say what these do. Example, I have no idea what a locus ID is right now.

Reply:
Done